plugins/services/src/js/service-configuration/PodGeneralConfigSection.js

```diff
@@ -78,67 +78,81 @@ module.exports = ({appConfig}) => {
-        </Row>
-        <Row>
-          <Label>Memory</Label>
-          <Value>{getContainerResourceSummary('mem', appConfig)}</Value>
-        </Row>
-        <Row>
-          <Label>Disk</Label>
-          <Value>{getContainerResourceSummary('disk', appConfig)}</Value>
-        </Row>
-        <Row>
-          <Label>GPU</Label>
-          <Value>{getContainerResourceSummary('gpu', appConfig)}</Value>
-        </Row>
+      <ConfigurationMapHeading level={1}>General</ConfigurationMapHeading>
+      <ConfigurationMapSection key="pod-general-section">
+        <ConfigurationMapRow>
+          <ConfigurationMapLabel>Service ID</ConfigurationMapLabel>
+          <ConfigurationMapValue value={appConfig.id} />
+        </ConfigurationMapRow>
+        <ConfigurationMapRow>
+          <ConfigurationMapLabel>Instances</ConfigurationMapLabel>
+          <ConfigurationMapValueWithDefault value={fields.instances} />
+        </ConfigurationMapRow>
+        <ConfigurationMapRow>
+          <ConfigurationMapLabel>CPU</ConfigurationMapLabel>
+          <ConfigurationMapValue>
+            {getContainerResourceSummary('cpus', appConfig)}
+          </ConfigurationMapValue>
+        </ConfigurationMapRow>
+        <ConfigurationMapRow>
+          <ConfigurationMapLabel>Memory</ConfigurationMapLabel>
+          <ConfigurationMapValue>
+            {getContainerResourceSummary('mem', appConfig)}
+          </ConfigurationMapValue>
+        </ConfigurationMapRow>
+        <ConfigurationMapRow>
+          <ConfigurationMapLabel>Disk</ConfigurationMapLabel>
+          <ConfigurationMapValue>
+            {getContainerResourceSummary('disk', appConfig)}
+          </ConfigurationMapValue>
+        </ConfigurationMapRow>
+        <ConfigurationMapRow>
+          <ConfigurationMapLabel>GPU</ConfigurationMapLabel>
+          <ConfigurationMapValue>
+            {getContainerResourceSummary('gpu', appConfig)}
+          </ConfigurationMapValue>
+        </ConfigurationMapRow>
         {fields.backoff && (
-          <Row>
-            <Label>Backoff</Label>
+          <ConfigurationMapRow>
+            <ConfigurationMapLabel>Backoff</ConfigurationMapLabel>
             <DurationValue
               units="sec"
               value={fields.backoff} />
-          </Row>
+          </ConfigurationMapRow>
         )}
         {fields.backoffFactor && (
-          <Row>
-            <Label>Backoff Factor</Label>
-            <Value value={fields.backoffFactor} />
-          </Row>
+          <ConfigurationMapRow>
+            <ConfigurationMapLabel>Backoff Factor</ConfigurationMapLabel>
+            <ConfigurationMapValue value={fields.backoffFactor} />
+          </ConfigurationMapRow>
         )}
         {fields.maxLaunchDelay && (
-          <Row>
-            <Label>Backoff Max Launch Delay</Label>
+          <ConfigurationMapRow>
+            <ConfigurationMapLabel>
+              Backoff Max Launch Delay
```

Comment:
This is what I was trying to avoid with `MapLabel` instead of `ConfigurationMapLabel` 😄